constants/__init__.py

```diff
@@ -27,13 +29,48 @@
 SUBNET_UID = 9
 # The root directory of this project.
 ROOT_DIR = Path(__file__).parent.parent
-# The maximum bytes for the hugging face repo (5 Gigabyte).
-MAX_HUGGING_FACE_BYTES = 5 * 1024 * 1024 * 1024
+# Block at which 7b models, 2048 sequence lengths, bfloat16, and flash attention are used.
+BLOCK_7B = 9_999_999
+SEQUENCE_LENGTH_1 = 1024
+SEQUENCE_LENGTH_2 = 2048
+# A mapping of block numbers to sequence length used for inference.
+SEQUENCE_LENGTHS = [(0, SEQUENCE_LENGTH_1), (BLOCK_7B, SEQUENCE_LENGTH_2)]
+# A mapping of block numbers to whether optimizations (bfloat16 and flash attention) are used.
+OPTIMIZATIONS_USED = [(0, False), (BLOCK_7B, True)]
+# A mapping of block numbers to the max model size as of that block.
+MAX_MODEL_BYTES = [
```

Comment:
Nit: Add a comment that this should be ordered by block?

Reply:
Done.